tests/standalone_tests.c

```diff
@@ -72,3 +72,35 @@ TEST_CASE("test missing placeholders", "[standalone]")
 	auto buf2 = std::string("the meaning of life is unknown");
 	REQUIRE(strcmp(buf.data(), buf2.data()) != 0); // these strings arent equal
 }
+
+TEST_CASE("test if null and nullptr are both valid for uninstantiated types", "[standalone]")
+{
```

Comment:
Added test to ensure nothing broke on the conversion. It can be removed or commented out later.